### PR TITLE
Imtac Config 2

### PR DESCRIPTION
**Pull request by AdamMcFet**

### CODE REVIEW

File: project/sites/imtac/settings.local.php [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?php
+
```
Comment (DuttonMa) [inline project/sites/imtac/settings.local.php:2]: This settings.local.php file should not be committed to git, so you'll need to remove it from this PR

**Review by DuttonMa — CHANGES_REQUESTED** [review]
Looks good, just the one change

File: project/sites/imtac/settings.local.php (hunk 1 above)
Comment (AdamMcFet) [reply project/sites/imtac/settings.local.php]: done, this is now removed from the commit

**Review by DuttonMa — APPROVED** [review]
Good to go now

**Pull request merged**